.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,66 @@
+---
+name: 🐛 Issue report
```

Comment:
Do we want this template to match the Flutter one? There seems to be stuff missing (e.g., development OS)

Reply:
This is only thing missed. I'll add it.